app/views/notifier/crop_request_rejected.html.haml

```diff
@@ -2,8 +2,11 @@
 %p Hello #{@member.login_name},
 
 %p
-  Your request for the new crop: #{link_to @crop.name, crop_url(@crop)} has been rejected for the following reason.
+  We're sorry, but your request for the new crop: #{link_to @crop.name, crop_url(@crop)} has been rejected for the following reason:
 
-  = @crop.reason_for_rejection
+  - if @crop.reason_for_rejection != "other"
+    = @crop.reason_for_rejection
+  - else
+    = @crop.rejection_notes
```

Comment:
How about encapsulating this little bit of logic into a convenience method on the model called something like `rejection_explanation`?